src/react/databrowser/objects/details/__tests__/Properties.test.tsx

```diff
@@ -322,8 +325,10 @@ describe('Properties', () => {
     const tableItems = component.find(T.Row);
     const seventh = tableItems.at(7);
     expect(seventh.find(T.Key).text()).toContain('Temperature');
-    expect(seventh.find(T.Value).text()).toContain(
+    expect(seventh.find(T.GroupValues).text()).toContain(
       'Restored (1 day remaining)',
     );
+    expect(seventh.find('button#restore-button').prop('label')).toBe('Restore');
+    expect(seventh.find('button#restore-button').prop('disabled')).toBe(true);
```

Comment:
This is testing implementation details through the id of the button 😭 but ok for now else it would be too costly

Reply:
Yes ..  I don't want to look at this test .. 🥲